pkg/plugins/builtin/echo/README.md

```diff
@@ -8,6 +8,7 @@ This plugin returns the output defined in its configuration, without performing
 |---|---
 | `output` | an object with the complete output of the step
 | `metadata` | an object containing the metadata returned by the step
+| `unmarshal` | defines whether unmarshal the output if it's a string or byte array before returning
```

Comment:
```suggestion
| `unmarshal` | if `true`, then `output` is expected to be a single string containing valid JSON; this is useful for generating an object via templating, since uTask can normally only apply templating on strings
```